hack/quickstart/init-master.sh

```diff
@@ -6,6 +6,7 @@ REMOTE_PORT=${REMOTE_PORT:-22}
 CLUSTER_DIR=${CLUSTER_DIR:-cluster}
 IDENT=${IDENT:-${HOME}/.ssh/id_rsa}
 SSH_OPTS=${SSH_OPTS:-}
+CSR_AUTOAPPROVE=${CSR_AUTOAPPROVE:-false}
```

Comment:
Nit: can you put the same warning that this shouldn't be set to true on production clusters.